Firebase/Messaging/FIRMessagingPendingTopicsList.m

```diff
@@ -65,10 +68,9 @@ - (instancetype)initWithCoder:(NSCoder *)aDecoder {
   }
 
   if (self = [self initWithAction:action]) {
-    NSSet *topics = [aDecoder decodeObjectForKey:kPendingTopicBatchTopicsKey];
-    if ([topics isKindOfClass:[NSSet class]]) {
-      _topics = [topics mutableCopy];
-    }
+    _topics = [aDecoder decodeObjectOfClasses:
+                     [NSSet setWithObjects:NSSet.class, NSMutableString.class, nil]
```

Comment:
Should it be `NSMutableSet` and `NSString`?

Reply:
Good catch.